webpack.test.config.js

```diff
@@ -13,6 +13,14 @@ module.exports = {
             'stage-2'
           ]
         }
+      },
```

Comment:
As in the other comment, if we use generic styles, we don't need the sass-loader.